frontend/src/components/transactions/ReceiptOutcomeRow.tsx

```diff
@@ -0,0 +1,256 @@
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+
+import * as T from "../../libraries/explorer-wamp/transactions";
+
+import { displayArgs } from "./ActionMessage";
+
+export interface Props {
+  sirReceiptHash: string;
+  receiptsObj: {};
+  receiptsInfoObj: {};
+  senderIsReceiverActions: T.RpcAction[];
+  senderIsReceiver: Boolean;
+}
+
+export default class extends React.Component<Props> {
```

Comment:
follow the name pattern and export without next js warning

Reply:
@icerove , and this one too